yt/frontends/amrex/data_structures.py

```diff
@@ -908,6 +903,12 @@ def _parse_header_file(self):
             dre = self.domain_right_edge.copy()
             dre[2] = 2.0 * np.pi
             self.domain_right_edge = dre
+        if self.geometry == "spherical" and self.dimensionality < 3:
+            dre = self.domain_right_edge.copy()
+            if self.dimensionality < 2:
+                dre[1] = np.pi
+            dre[2] = 2.0 * np.pi
```

Comment:
nit, but I find the reverse order easier to read
```suggestion
            dre[2] = 2.0 * np.pi
            if self.dimensionality < 2:
                dre[1] = np.pi
```